From af2579adcada985eb33a329c99626c7b9170ecab Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sat, 25 May 2024 17:06:53 +0200
Subject: [PATCH 376/463] arm64: dts: rk3399-pinephone-pro: World cam doesn't
 have powerdown input

It's in the schematic, but does nothing.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index f2437625226d2..1f1e424a09ad4 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -760,7 +760,7 @@ wcam: camera@1a {
 		compatible = "sony,imx258";
 		reg = <0x1a>;
 		pinctrl-names = "default";
-		pinctrl-0 = <&wcam_rst &wcam_pdn>;
+		pinctrl-0 = <&wcam_rst>;
 
 		clocks = <&cru SCLK_CIF_OUT>;
 		clock-names = "xvclk";
@@ -770,7 +770,6 @@ wcam: camera@1a {
 		/*XXX: also depends on vcca1v8_codec for I2C bus power (currently always on) */
 
 		reset-gpios = <&gpio1 RK_PA0 GPIO_ACTIVE_LOW>;
-		powerdown-gpios = <&gpio2 RK_PD4 GPIO_ACTIVE_HIGH>;
 
 		orientation = <1>;
 		rotation = <270>;
@@ -1036,10 +1035,6 @@ wcam_rst: wcam-rst {
 			rockchip,pins = <1 RK_PA0 RK_FUNC_GPIO &pcfg_pull_none>;
 		};
 
-		wcam_pdn: wcam-pdn {
-			rockchip,pins = <2 RK_PD4 RK_FUNC_GPIO &pcfg_pull_none>;
-		};
-
 		ucam_rst: ucam-rst {
 			rockchip,pins = <1 RK_PA4 RK_FUNC_GPIO &pcfg_pull_none>;
 		};
-- 
2.46.0

